sdk/keyvault/azure-security-keyvault-administration/CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Release History
 
-## 4.2.0 (2022-09-09)
+## 4.2.0 (2022-09-19)
```

Comment:
Nit: could be 9-20. Depends if you're releasing tonight. Given everything goes live tomorrow one way or another, that's why I chose it. Not a big deal, though.